lightning/src/routing/mod.rs

```diff
@@ -26,7 +26,14 @@ use sync::{Mutex, MutexGuard};
 pub trait Score {
 	/// Returns the fee in msats willing to be paid to avoid routing through the given channel
 	/// in the direction from `source` to `target`.
-	fn channel_penalty_msat(&self, short_channel_id: u64, source: &NodeId, target: &NodeId) -> u64;
+	///
+	/// The maximum amount which may be in the HTLC at the given channel is given by
```

Comment:
Sentence is a little hard to parse. Would it be simpler to drop it and instead update the previous sentence to include the variable? (i.e., Returns the fee in msats willing to be paid to avoid routing `send_amt_msat` through ...)